bioconda_utils/utils.py

```diff
@@ -482,9 +482,7 @@ def get_inner_deps(dependencies):
     for meta, recipe in metadata:
         name = meta["package"]["name"]
         dag.add_edges_from((dep, name)
-                           for dep in set(get_inner_deps(chain(
-                               get_deps(meta, "build"),
-                               get_deps(meta, "host")))))
+                           for dep in set(get_inner_deps(get_deps(meta, "host"))))
```

Comment:
I get the failing case (dependency circle on those Perl recipes), but I don't think this change makes sense. Can't we just blacklist those recipes?